tests/proxy.rs

```diff
@@ -4,14 +4,15 @@ extern crate reqwest;
 mod support;
 
 #[test]
+#[ignore]
```

Comment:
We need `req.set_proxy(bool)` equivalent to get this working.

Reply:
The way to set that a request is proxied has changed in hyper 0.12. Instead, reqwest's `Connect` would need to set that a proxy is used when returning a [`Connected`](https://docs.rs/hyper/0.12.*/hyper/client/connect/struct.Connected.html#method.proxy).